### DISCUSSION TITLE
Lettera al Governo: il Recovery Plan per l'informatica e l'educazione aperte

### DISCUSSION THREAD

**Discussion by Shamar** [iisgiua/giuaschool#483] [Idee e proposte]
Salve, volevo segnalarvi questa bella lettera al Governo del Professor Angelo Raffaele Meo, Presidente della Associazione Software Libero:

https://www.softwarelibero.it/lettera-al-governo-del-prof-angelo-raffaele-meo

Personalmente, credo che sarebbe fantastico se il Governo spingesse le scuole ad adottare software libero per i Registri Elettronici delle scuole.

**Comment by trinko** [comment]
Grazie per la segnalazione. 
Personalmente non sono molto ottimista sulla questione, ma non si sa mai...
